packages/in_app_purchase/in_app_purchase_android/README.md

```diff
@@ -24,6 +24,6 @@ If you would like to contribute to the plugin, check out our
 [contribution guide](https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md).
 
 
-[1]: ../in_app_purchase/in_app_purchase
+[1]: ../in_app_purchase
```

Comment:
Relative links won't work on pub.dev so let's update this to:
```suggestion
[1]: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase/in_app_purchase
```

Reply:
Thanks for the advice.

I fixed in 
2a81932